src/core/processing/qgsprocessingparametertininputlayers.h

```diff
@@ -65,10 +65,13 @@ class CORE_EXPORT QgsProcessingParameterTinInputLayers: public QgsProcessingPara
     static QString typeName() { return QStringLiteral( "tininputlayers" ); }
 };
 
+///@cond PRIVATE
```

Comment:
I'd sip skip this too -- it was never supposed to be public api, and there's no need for python code to be using it so no danger in removing it now.

Reply:
Done.